spec/transition_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe ProMotion::Menu::Transition do
+
+  before do
+    @object = Object.new
+    @object.extend(ProMotion::Menu::Transition)
+
+    @delegate = UIApplication.sharedApplication.delegate
+    @left = LeftNavScreen.new
+    @right = RightNavScreen.new
+    @content = BlankScreen.new
+  end
+
+
+  describe "#visualStates" do
```

Comment:
Thanks for including specs! :smile: This one is pretty difficult to test too. Can you change this to:
```ruby
describe "#mask_for_transition" do
```
and include something like:
```ruby
describe "#transition=" do

  it "calls setDrawerVisualStateBlock" do
    @object.mock! :"setDrawerVisualStateBlock:" do |mask|
      mask.is_a?(Proc).should == true
    end
    @object.transition_animation = :slide
  end

end
```